interface-definitions/protocols-pim.xml.in

```diff
@@ -41,8 +57,134 @@
+            <properties>
+              <help>Enable PIM ECMP</help>
+            </properties>
+            <children>
+              <leafNode name="rebalance">
+                <properties>
+                  <help>Enable PIM ECMP Rebalance</help>
+                  <valueless/>
+                </properties>
+              </leafNode>
+            </children>
+           </node>
+           <node name="igmp">
+            <properties>
+              <help>Internet Group Management Protocol (IGMP) options</help>
+            </properties>
+            <children>
+              <leafNode name="watermark-warn">
+                <properties>
+                  <help>Configure group limit for watermark warning</help>
+                  <valueHelp>
+                    <format>u32:1-65535</format>
+                    <description>Group count to generate watermark warning</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-65535"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+            </children>
+           </node>
+           <leafNode name="join-prune-interval">
+            <properties>
+              <help>Join Prune Send Interval</help>
+              <valueHelp>
+                <format>u32:1-65535</format>
+                <description>Interval in seconds</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-65535"/>
+              </constraint>
+            </properties>
+            <defaultValue>60</defaultValue>
+          </leafNode>
+          <leafNode name="keep-alive-timer">
+            <properties>
+              <help>Keep alive Timer</help>
+              <valueHelp>
+                <format>u32:1-65535</format>
+                <description>Keep alive Timer in seconds</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-65535"/>
+              </constraint>
+            </properties>
+            <defaultValue>210</defaultValue>
+          </leafNode>
+          <leafNode name="packets">
+            <properties>
+              <help>Packets to process at onece</help>
+              <valueHelp>
+                <format>u32:1-255</format>
+                <description>Number of packets</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-255"/>
+              </constraint>
+            </properties>
+            <defaultValue>3</defaultValue>
+          </leafNode>
+          <leafNode name="register-accept-list">
+            <properties>
+              <help>Only accept registers from a specific source prefix list</help>
+              <valueHelp>
+                <format>txt</format>
+                <description>Prefix-list to apply</description>
```

Comment:
We should probably use "prefix list" without a hyphen when it's a generic term rather than a CLI keywords.

Reply:
```
(08:05) cpo lnx01:~/vyos-1x [frr-pim-T5733] # git grep "<description>" | grep "prefix list" | wc -l
0
(08:06) cpo lnx01:~/vyos-1x [frr-pim-T5733] # git grep "<description>" | grep "prefix-list" | wc -l
11
```

`prefix-list` is the more common used phrase accross VyOS - I tend to rather keep it consistent on all places